controllers/depresolver/depresolver_test.go

```diff
@@ -1390,6 +1415,23 @@ func TestNsServerNamesWithMultipleExtClusterGeoTag(t *testing.T) {
 	}
 }
 
+func TestNsServerNamesForLocalEdgeDNS(t *testing.T) {
+	// arrange
+	defer cleanup()
+	for _, edgeDNSServer := range []string{"127.0.0.1", "localhost"} {
+		customConfig := predefinedConfig
+		customConfig.EdgeDNSServer = edgeDNSServer
+		configureEnvVar(customConfig)
+		resolver := NewDependencyResolver()
+		// act
+		config, err := resolver.ResolveOperatorConfig()
+		// assert
+		assert.NoError(t, err)
+		assert.Equal(t, config.GetClusterNSName(), edgeDNSServer)
+		assert.True(t, reflect.DeepEqual(config.GetExternalClusterNSNames(), map[string]string{"uk": edgeDNSServer, "eu": edgeDNSServer}))
```

Comment:
`"uk"` looks unexpected. Typo?

Reply:
https://github.com/k8gb-io/k8gb/pull/505/files#diff-384d90ba8267cb88a0aab2845e0778c734551e380ce8040b033b006584153fdeR46 - hm. I didn't realize we check to Brexit in our tests :D